apps/design/frontend/src/system_settings_screen.tsx

```diff
@@ -313,35 +313,36 @@ export function SystemSettingsForm({
                 />
               </InputGroup>
             )}
-            {features.MINIMUM_DETECTED_SCALE_OPTION && (
```

Comment:
I was previously feature flag gating this because I didn't want SLI turning this on in v4.0.2, which as noted earlier has an implementation of this check that we weren't 100% confident in. But now safe to remove the feature flag.